src/pages/Tournament/TournamentList/parts.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
```

Comment:
Codacy has a fix for the issue: [Replace `React,·{·useState·}·from·'react` with `{·Button,·Table·}·from·'antd`](https://app.codacy.com/gh/othello-organisation-singapore/othello-storm-system-frontend/pullRequest?prid=8062249)
```suggestion
import { Button, Table } from 'antd';
```